packages/server-wallet/src/wallet/multi-threaded-wallet/manager.ts

```diff
@@ -57,6 +60,14 @@ export class WorkerManager {
     });
   }
 
+  /**
+   *
+   * @param {operation, args}: operation & arguments to send to worker
+   *
+   * Sends {operation, args} to a worker thread.
+   * RESOLVES WHEN: the worker thread sends a "right" message containing a result
```

Comment:
The resolve/reject info might be good to include in a `@returns` tag?

Reply:
Is that the expected use of `@returns`? I am unfamiliar with TSDoc.

Reply:
`@returns` specifies what a function returns. 

So we could have something like `* @returns a promise that resolves when ....`

I don't think it really matters too much though! I'm happy that this information is in TSDoc in some manner.

Reply:
I'll switch it to `@returns`. I think TSDoc/we can do better, with a `@resolves` keyword, but I don't know how feasible that it.

Reply:
There is some discussion about that here https://github.com/Microsoft/tsdoc/issues/63

Reply:
b9fd23069f2cd6893d27b68a1aefe7225a457095